src/harness/projectsRunner.ts

```diff
@@ -422,10 +422,16 @@ class ProjectRunner extends RunnerBase {
         }
 
         function getErrorsBaseline(compilerResult: CompileProjectFilesResult) {
-            const inputFiles = ts.map(compilerResult.configFileSourceFiles.concat(
-                compilerResult.program ?
-                    ts.filter(compilerResult.program.getSourceFiles(), sourceFile => !Harness.isDefaultLibraryFile(sourceFile.fileName)) :
-                    []),
+            const inputSourceFiles = compilerResult.configFileSourceFiles.slice();

```

Comment:
How does this change involve `ReadonlyArray`? What necessitated the change in style?